docs/README.md

```diff
@@ -1,46 +1,50 @@
-* **[Caikit-nlp](https://github.com/opendatahub-io/caikit-nlp)**: Caikit module that handles NLP style models
-* **[KServe](https://github.com/opendatahub-io/kserve)**: Orchestrates model serving for all types of models, servingruntimes implement loading given types of model servers. KServe handles the lifecycle of the deployment object, storage access, networking setup, etc.
-* **[Service Mesh](https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/ossm-architecture.html)** (istio): Service mesh networking layer, manages traffic flows, enforces access policies, etc.
-* **[Serverless](https://docs.openshift.com/serverless/1.29/about/about-serverless.html)** (knative): Allows for serverless deployments of models
+The Caikit-TGIS-serving image consists of these components:
+* **[Text Generation Inference Server (TGIS)](https://github.com/opendatahub-io/text-generation-inference)**: The serving backend that loads the models and provides the inference engine.
+* **[Caikit](https://github.com/opendatahub-io/caikit)**: A wrapper layer that handles the lifecycle of the TGIS process, provides the inference endpoints, and has modules to handle different model types.
+* **[Caikit-nlp](https://github.com/opendatahub-io/caikit-nlp)**: The Caikit module that handles Natural Language Processing (NLP)-style models.
+* **[KServe](https://github.com/opendatahub-io/kserve)**: A Kubernetes Custom Resource Definition that orchestrates model serving for all types of models. It includes serving runtimes that implement the loading of given types of model servers. KServe handles the lifecycle of the deployment object, storage access, and networking setup.
+* **[Service Mesh](https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/ossm-architecture.html)** (istio): The service mesh networking layer that manages traffic flows and enforces access policies.
+* **[Serverless](https://docs.openshift.com/serverless/1.29/about/about-serverless.html)** (knative): A cloud-native development model that allows for serverless deployments of data models.
+
+## Architecture of the stack
+
+![KServe+Knative+Istio+Caikit_TGIS Diagram](https://github.com/opendatahub-io/caikit-tgis-serving/assets/8479010/7009b95d-0f6f-4f18-b0e6-355f360a5ad1)
 
 
 ## Installation
-### Prerequisites
-- Openshift Cluster 
-  - This doc is written based on a ROSA cluster and has been tested with an OSD cluster as well
-  - Many of the tasks in this tutorial require cluster-admin permission level (e.g., install operators, set service-mesh configuration, enable http2, etc)
-  - 4 CPU and 16 GB memory in a node for inferencing (can be adjusted in servingRuntime deployment)
-- CLI tools
-  - oc cli
 
+*Note:* The procedures for installing and deploying the Caikit-TGIS-serving image are based on a Red Hat OpenShift Service for AWS (ROSA) cluster. They have not been tested on an OpenShift Dedicated cluster.
```

Comment:
It actually has been tested with OSD and is works fine. We can maybe say, the installation procedures have been tested on ROSA and OSD clusters.